src/constants/index.ts

```diff
@@ -1362,6 +1434,45 @@ export const POOLS_MAP: PoolsMap = {
     route: "tbtc-evmosbtc",
     rewardPids: buildPids({}),
   },
+  [TBTC_METAPOOL_V2_BTCV2_V3_NAME]: {
+    name: TBTC_METAPOOL_V2_BTCV2_V3_NAME,
+    lpToken: TBTC_SWAP_V3_TOKEN,
+    poolTokens: TBTC_POOL_TOKENS,
+    addresses: TBTC_META_SWAP_V3_DEPOSIT_ADDRESSES,
+    isSynthetic: true,
+    type: PoolTypes.BTC,
+    metaSwapAddresses: TBTC_META_SWAP_V3_ADDRESSES,
+    underlyingPoolTokens: TBTC_UNDERLYING_POOL_TOKENS,
+    underlyingPool: BTC_POOL_V2_NAME,
+    route: "tbtcv2btcv2v3",
+    rewardPids: buildPids({ [ChainId.MAINNET]: 5 }),
```

Comment:
double check on chain, but the last pid was 7 and this is new, so I'm guessing this is 8
https://github.com/saddle-finance/multisig-scripts/pull/11/files